tests/rules/javascript/test.no_unsafe_innerhtml.js

```diff
@@ -1,5 +1,9 @@
 import { VALIDATION_WARNING } from 'const';
 import JavaScriptScanner from 'scanners/javascript';
+import {
+  UNSAFE_DYNAMIC_VARIABLE_ASSIGNMENT,
+  NO_DOCUMENT_WRITE } from 'messages';
```

Comment:
Looks like there are a bunch of imports like this in the code base so these can be ignored.